src/components/grid.vue

```diff
@@ -146,7 +167,7 @@ export default {
   async mounted() {
     // Listen for board state changes
     gamesCollection
-      .doc('VUqGnWBLmgulz3X5O13h') // Obtain from state in the future when rooms are implemented
+      .doc(this.currentGame) // Obtain from state in the future when rooms are implemented
```

Comment:
anticipating many merge conflicts with the pregame on this file based on changes